explorer/src/lib/components/block-details/BlockDetails.svelte

```diff
@@ -0,0 +1,170 @@
+<svelte:options immutable={true} />
+
+<script>
+  import { mdiArrowLeft, mdiArrowRight } from "@mdi/js";
+  import { AppAnchor, ListItem } from "$lib/components";
+  import { Card, Icon, ProgressBar, Switch } from "$lib/dusk/components";
+  import { createCurrencyFormatter } from "$lib/dusk/currency";
+  import {
+    calculateAdaptiveCharCount,
+    getRelativeTimeString,
+    middleEllipsis,
+  } from "$lib/dusk/string";
+  import { onMount } from "svelte";
+  import "./BlockDetails.css";
+
+  const formatter = createCurrencyFormatter("en", "DUSK", 0);
+  const blockHeight = 34526;
+
+  /** @type {number} */
+  let screenWidth = window.innerWidth;
+
+  /** @type {boolean} */
+  let togglePayload = false;
```

Comment:
Should this be `isPayloadToggled` or something similar? `togglePayload` sounds more like a function.